### PR TITLE
fix 473 for fs.WriteFile 

### PR DESCRIPTION
**Pull request by ksandhu16**

### CODE REVIEW

**Comment by VictorKubrak** [comment]
This test seems to already exist: https://github.com/filerjs/filer/blob/master/tests/spec/fs.writeFile-readFile.spec.js
But if you still want to write this test, try adding data parameter to your writeFile().
Look https://nodejs.org/api/fs.html#fs_fs_writefile_file_data_options_callback for method description.

File: tests/spec/fs.writeFile-readFile.spec.js [hunk 1]
```diff
@@ -99,3 +99,25 @@ describe('fs.writeFile, fs.readFile', function() {
     });
   });
 });
+
+describe('fsPromises.Write', function() {
+  it('should write, read a utf8 file without specifying utf8 in writeFile', function() {
+    //var fs = util.fs();
+    //var contents = 'This is a file.';
```
Comment (humphd) [inline tests/spec/fs.writeFile-readFile.spec.js]: You don't want to leave commented code in a PR like this.  If you don't need the code, remove it.  There's no need to keep it around, since we can go back in time with git to recover old code if necessary.

File: tests/spec/fs.writeFile-readFile.spec.js [hunk 2]
```diff
@@ -99,3 +99,25 @@ describe('fs.writeFile, fs.readFile', function() {
...
+	
+    var fsPromises = util.fs().promises;
+	
+    return fsPromises.writeFile('/myfile' , { encoding: 'utf8' })
```
Comment (humphd) [inline tests/spec/fs.writeFile-readFile.spec.js]: `writeFile` needs an argument with the `data` to write

File: tests/spec/fs.writeFile-readFile.spec.js [hunk 3]
```diff
@@ -99,3 +99,25 @@ describe('fs.writeFile, fs.readFile', function() {
...
+  it('should write, read a utf8 file without specifying utf8 in writeFile', function() {
+    //var fs = util.fs();
+    //var contents = 'This is a file.';
+	
+    var fsPromises = util.fs().promises;
+	
+    return fsPromises.writeFile('/myfile' , { encoding: 'utf8' }) 
+      
+      .then(() => {
+        return  fsPromises.readFile('/myfile', 'utf8')
```
Comment (humphd) [inline tests/spec/fs.writeFile-readFile.spec.js]: You can shorten this to:

```js
.then(() => fsPromises.readFile('/myfile', 'utf8'))
.then(data => {
  expect(data).to.equal(contents); // here assuming you have some kind of data contents defined above
});
```

File: tests/spec/fs.writeFile-readFile.spec.js [hunk 4]
```diff
@@ -99,3 +99,25 @@ describe('fs.writeFile, fs.readFile', function() {
...
+          
+          .catch((error) => {
```
Comment (humphd) [inline tests/spec/fs.writeFile-readFile.spec.js]: This isn't necessary

File: tests/spec/fs.writeFile-readFile.spec.js [hunk 5]
```diff
@@ -99,3 +99,25 @@ describe('fs.writeFile, fs.readFile', function() {
...
+		
+            expect(error).not.to.exist;
+            //expect(data).to.equal(contents);
```
Comment (humphd) [inline tests/spec/fs.writeFile-readFile.spec.js]: Not sure why these are commented out?

File: tests/spec/fs.writeFile-readFile.spec.js [hunk 6]
```diff
@@ -99,3 +99,25 @@ describe('fs.writeFile, fs.readFile', function() {
...
+            //done();
+          });
+      });
+  });
+});
```
Comment (humphd) [inline tests/spec/fs.writeFile-readFile.spec.js]: Always include a newline at the end of each file.

**Comment by humphd** [comment]
NOTE: when you create a PR, make sure you provide both a Subject line and a full Description to describe what the changes do, why you did what you did, which Issue # it fixes (e.g., Fixes #473).
